documentation/release_notes.rst

```diff
@@ -53,6 +53,8 @@ New in This Release
   with ``unseq`` or ``par_unseq`` policy when compiled by Intel® oneAPI DPC++/C++ Compiler
   with ``-fiopenmp``, ``-fiopenmp-simd``, ``-qopenmp``, ``-qopenmp-simd`` options on Linux.
   To avoid the issue, pass ``-fopenmp`` or ``-fopenmp-simd`` option instead.
+- Incorrect results may be produced with group reduce operation through ``std::reduce``
```

Comment:
Does the issue reproduce with other reduce-based algorithms, e.g. `transform_reduce`?
Please, note that other algorithms are referenced without namespaces, so I would suggest omitting `std::` part.